.nuget/create_nuget.py

```diff
@@ -36,9 +40,44 @@
         <dependencies> </dependencies>
     </metadata>
         <files>
+        <file src="build\**" target="build"/>
         <file src="runtimes\**" target="runtimes"/>
         </files>
     </package>
     ''' % version
+    prop_str = '''
+    <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup Condition="Exists('packages.config') OR
+                            Exists('$(MSBuildProjectName).packages.config') OR
+                            Exists('packages.$(MSBuildProjectName).config')">
+        <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\*.dll"
+                Condition="'$(PlatformTarget)' == 'x64'">
+        <Content Include="$(MSBuildThisFileDirectory)\..\runtimes\win-x64\native\*.exe"
```

Comment:
I'm not sure why `.exe` needs to be copied to the output directory.  But if you have user scenarios for it, then I guess it should be.  Is this really needed?